mono/metadata/metadata.c

```diff
@@ -6638,3 +6638,36 @@ mono_method_get_wrapper_cache (MonoMethod *method)
 		return &method->klass->image->wrapper_caches;
 	}
 }
+
+#if CHECKED_BUILD
```

Comment:
We should try to avoid this sort of conditional compilation creep.

It makes for a more fragile code base, say I break it tomorrow, we won't pick that
until that until the next time we do a checked build.

In addition to that, now it's one extra configuration to keep in mind when casually looking at that code.

The nice thing is that we don't actually need to worry about possible extra bloat from this function because it has non-public visibility and the linker will remove it.